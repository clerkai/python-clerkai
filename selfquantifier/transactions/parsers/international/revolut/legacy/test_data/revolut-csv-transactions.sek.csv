Completed Date;Reference;Paid Out (SEK);Paid In (SEK);Exchange Out;Exchange In; Balance (SEK);Exchange Rate;Category
22. Oct 2020;Payment from WOLLSEN,FREDRIK;;10,00 ;;;10,00 ; ;General
