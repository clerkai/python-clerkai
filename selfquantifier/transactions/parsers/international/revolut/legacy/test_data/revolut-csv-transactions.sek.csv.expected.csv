Real Date,Bank Date,Payee,Bank Message,Amount,Currency,Balance,Foreign Currency,Foreign Currency Amount,Foreign Currency Rate,Original data,Raw Real Date,Raw Bank Date,Raw Payee,Raw Bank Message,Raw Amount,Raw Currency,Raw Balance,Raw Foreign Currency,Raw Foreign Currency Amount,Raw Foreign Currency Rate
,2020-10-22,,"Payment from WOLLSEN,FREDRIK",10.00,SEK,10.00,,,,"{'Completed Date': '22. Oct 2020', 'Reference': 'Payment from WOLLSEN,FREDRIK', 'Paid Out (SEK)': nan, 'Paid In (SEK)': '10,00\xa0', 'Exchange Out': nan, 'Exchange In': nan, ' Balance (SEK)': '10,00\xa0', 'Exchange Rate': '\xa0', 'Category': 'General'}",,22. Oct 2020,,"Payment from WOLLSEN,FREDRIK","10,00 ",SEK,"10,00 ",,,
